### DISCUSSION TITLE
Choo-Choo, LittleJS is here, all aboard! 🚂

### DISCUSSION THREAD

**Discussion by KilledByAPixel** [KilledByAPixel/LittleJS#6] [Announcements]
## 👋 Hi everyone! 🚂

I am so excited to share LittleJS with you all! This is a passion project for me to make a game engine that is powerful, versatile, and small. I cant wait to see what you all make with it.

Please use this forum to...

* Ask questions about LittleJS
* Share ideas for new features
* Post games you made with the engine

To get started, comment below with an introduction of yourself and tell us about what kind of games you want to make!

![preview2](https://user-images.githubusercontent.com/16227600/135759753-1d2c9d56-e5c8-4e59-9c66-1a2db0c71262.jpg)

**Comment by alpacachen** [comment]
hi，do you need Chinese translation for your site?it's an honor for me to do this

**Reply by KilledByAPixel** [reply]
Thank you so much for offering. I would love to have your help with that, but not just yet as things are still in flux. But yeah I will get back to you when I am ready if that is ok with something for you to translate. I appreciate it, thanks!

**Comment by AxeLanderMoreira** [comment]
Hi there. 
I've learned about LittleJS in a post you've made on the gamedev subreddit, but just now I have the opportunity and spare time to give it a try.
I am interested on how your library would fit into other HTML5-based devices and environments (SmartTV / DigitalTV mainly). It definitely looks promising. 
My main goal, however, is to make some simple games for my kid, and show him a few things about amateur game development. :)

**Reply by KilledByAPixel** [reply]
That's great! I think it should work on anything that supports HTML5 but I have yet to try on a SmartTV. I think your kid will love the games you make with this. :)
